### DISCUSSION TITLE
Testing DOKAN_OPTION_CURRENT_SESSION feature

### DISCUSSION THREAD

**Discussion by israelzimmerman** [dokan-dev/dokany#1226] [General]
I have tested current session feature and it does not working at all, checked with ssh server on the computer and connected using ssh client from other computer

I need to prevent unauthorized users to use my file system directory.
After checking on the driver sources i saw that the message "Not using Mount Manager" should appear on System events 
and it does not

I suggest to add more debug information including the following parameters:
Dcb->UseMountManager
Dcb->MountGlobally
SessionId

I am using the mirror sample with this command line:
mirror.exe /c /r C:\mount\fs.mirror /l C:\mount\fs

Do you have any ideas why current session mode is not working?

I think that some problems also exists after computer sleep that need to be checked

**Comment by Liryna** [comment]
Hi @israelzimmerman ,

`DOKAN_OPTION_CURRENT_SESSION` only works for drives mounted with a letter.
Here you are mounting in a folder which the second user has access to. Solution is to either mount in a folder under your home (which the other does not have access to) or mount with a restricted `DOKAN_OPTIONS::VolumeSecurityDescriptor`.

> After checking on the driver sources i saw that the message "Not using Mount Manager" should appear on System events

This is only logged for mount with drive letter.

I created #1227 to avoid this usage.
